CME & SUclass Theme Stage Testing Changes

caesar2164 opened the pull request:
@stvstnfrd & @kluo 

I'll add all the commits here

Commit:
Enables "Register" link in CME header

- Also simplifies CME header link code

caesar2164 commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Fix Register Button for CME](https://trello.com/c/zN53fcnS/199-fix-register-button-for-cme)

caesar2164 commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Fix Register Button for CME](https://trello.com/c/zN53fcnS/199-fix-register-button-for-cme)

Commit:
Ensures SUclass header has SYSADMIN superuser link

- Also simplifies conditional code

caesar2164 commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Confirm that SYSADMIN link is present on suclass](https://trello.com/c/tVsm0zWr/202-confirm-that-sysadmin-link-is-present-on-suclass)

stvstnfrd commented on cme/lms/templates/header.html:
```diff
@@ -12,18 +12,21 @@
+        %endif
+    </%block>
+% else:
+    <%block name="navigation_global_links">
```
Have you taken a look at `./lms/templates/navigation.html` in the platform [1]?

It looks like the register link is originally set in `navigation_other_global_links`, not `navigation_global_links`.

I think the default logic actually gives us what we want.
Assuming so, let's remove this override and drop the one below too.

[1] https://github.com/Stanford-Online/edx-platform/blob/master/lms/templates/navigation.html#L154-L171

stvstnfrd commented on cme/lms/templates/header.html:
```diff
@@ -12,18 +12,21 @@
+    </%block>
 %endif
+
+<%block name="navigation_other_global_links"></%block>
```
See note above about removing.

stvstnfrd commented on cme/lms/templates/header.html:
```diff
@@ -12,18 +12,21 @@
-    <%block name="navigation_global_links"></%block>
-    <%block name="navigation_other_global_links"></%block>
+% if UserProfile.has_registered(user):
+    <%block name="navigation_global_links_authenticated">
```
Similarly, I think the default value suffices here.

Let's try unforking.

caesar2164 replied:
We can't cause platform `About` isn't configurable through settings.

stvstnfrd replied:
Let's just wrap it in a feature flag and make it configurable then :)

stvstnfrd commented on suclass/lms/templates/header.html:
```diff
@@ -12,18 +12,16 @@
-%else:
-    <%block name="navigation_global_links"></%block>
-    <%block name="navigation_other_global_links"></%block>
+% if UserProfile.has_registered(user):
```
Let's try unforking all of these blocks too.
Most links are already conditional based on feature flags.

stvstnfrd commented on suclass/lms/templates/header.html:
```diff
@@ -12,18 +12,16 @@
   <img src="${static.url('images/stanford-S-logo.png')}" height="35" alt="Home Page" />
```
Thanks to comp theming, we may be able to unfork this block.

The template tries to pull the logo from a number of locations first,
but falls back to `images/logo.png`.
I think if we renamed the file to the above, it should be found automatically.

stvstnfrd commented:
Hrmm, if we can unfork each of these mentioned blocks, we can actually remove this file entirely :)

Commit:
Updates theme images

- Removes unused images
- Replaces old images with current ones
- Removes logo overriding block for CME and SUclass

Commit:
Remove header.html forks

stvstnfrd commented:
+1 after you test locallly
